example_projects/person.py

```diff
@@ -0,0 +1,22 @@
+"""
+The person class
+"""
+
+
+class Person:
+    """
+    Represents a person
+    """
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+
+    def introduce(self):
+        """
+        Produce a short intro.
+        """
+        return f"Hi, my name is {self.name} and I am {self.age} years old."
+
+    def __repr__(self):
+        return f"[name': {self.name} 'age': {self.age}]"
```

Comment:
Might be better to use curly braces instead of bracket, as everything in python is a dict.

Reply:
I think you meant JSON format; resolved